src/pybind/mgr/volumes/fs_subvolume.py

```diff
@@ -0,0 +1,362 @@
+"""
+Copyright (C) 2019 Red Hat, Inc.
+
+LGPL2.1.  See file COPYING.
+"""
+
+import errno
+import logging
+import os
+
+import cephfs
+import rados
+
+def to_bytes(param):
+    '''
+    Helper method that returns byte representation of the given parameter.
+    '''
+    if isinstance(param, str):
+        return param.encode()
+    else:
+        return str(param).encode()
```

Comment:
`str(str(...))` is idempotent so I don't really see a reason to check the type of `param`.

Reply:
Done.